.github/ISSUE_TEMPLATE/qa_issue_template.md

```diff
@@ -0,0 +1,24 @@
+---
+name: Manual QA
+about: Use this template for running manual QA issues.
+title: "[DATE]: [FEATURE NAME]"
+labels: qa, review
+---
+# Manual QA for # 
```

Comment:
should be some clarification as comment what should I enter here. e.g. [DVC issue template](https://raw.githubusercontent.com/iterative/dvc/main/.github/ISSUE_TEMPLATE/bug_report.md)